pyproject.toml

```diff
@@ -20,15 +19,18 @@ classifiers = [
 ]
 
 [tool.poetry.plugins."sceptre.resolvers"]
-"custom" = "resolver.custom:Custom"
 
 [tool.poetry.dependencies]
 python = ">=3.8,<3.12"
+types-PyYAML = "6.0.12.9"
 
 [tool.poetry.group.dev.dependencies]
+autopep8 = "^2.0.2"
```

Comment:
where is this used?

Reply:
pre-commit does all the linting.  it is already setup with `black` which basically does the same thing as autopep8.  you can just remove this dependency

Reply:
Ok done